src/pages/get-started.tsx

```diff
@@ -0,0 +1,287 @@
+        >
+            <ContentSection>
+                <Tabs
+                    activeTabKey={activeTab || 'install_app'}
+                    tabsWrapperClassName="md:flex md:gap-8"
+                    navClassName="flex sm:flex-row md:flex-col md:!border-none md:gap-6 md:!justify-start"
+                    contentClassName="!py-0"
+                    tabs={[
+                        {
+                            key: 'install_app',
+                            title: (
+                                <>
+                                    <div className="hidden items-center md:flex">
+                                        <div className="flex-col gap-y-2">
+                                            <Heading size="h4">
+                                                Install the app <Badge text="Beta" size="small" color="blurple" />
+                                            </Heading>
+                                            <p
+                                                className={classNames(
+                                                    'm-0 text-base font-normal !text-gray-500',
+                                                    activeTab === 'install_app' && '!text-gray-200'
+                                                )}
+                                            >
+                                                Share Sourcegraph with your team by requesting a cloud or self-hosted
+                                                trial.
+                                            </p>
+                                        </div>
+                                        <CheckCircleIcon
+                                            className={classNames(
+                                                'h-[17px] rounded-full bg-white p-[3px]  text-black',
+                                                activeTab !== 'install_app' && 'opacity-0'
+                                            )}
+                                        />
+                                    </div>
+
+                                    <div className="mb-2 flex gap-x-2 font-normal md:hidden">
+                                        Sourcegraph App <Badge text="Beta" size="small" color="blurple" className="" />
+                                    </div>
+                                </>
+                            ),
+                            content: <InterstitialAppContent />,
+                            className: classNames(
+                                'md:!p-4 md:!border  md:rounded-[5px] md:max-w-[364px] md:!text-left',
+                                activeTab === 'install_app'
+                                    ? 'md:sg-bg-linear-space md:text-white md:shadow-xl'
+                                    : 'md:!border-gray-300'
+                            ),
+                        },
+                        {
+                            key: 'entrerprise_trial',
```

Comment:
Before we merge this, I want to draw attention to the fact that `entrerprise_trial` is the name given to the tab here, so the final direct URL to this tab will be `https://about.sourcegraph.com/get-started?app=entrerprise_trial`

If this is merged as-is with `entrerprise_trial` as the tab name, we have to make sure that we're using that final URL and that we're not using different versions or spellings of this URL anywhere, such as ending in `?app=enterprise` (the link won't work) or ending in `?app=enterprise_trial` (the link won't work; note the spelling difference).

Also, if this `entrerprise_trial` key gets changed to something else for any reason, then we need to know about the change beforehand because we're hardcoding this link URL in places where we can't easily change it after the release, such as in the Sourcegraph App. 🙂

Reply:
Ah thanks @marekweb for pointing this out. it's a typo and would be resolved shortly 🙏.